addons/mail/tests/test_mail_composer.py

```diff
@@ -264,6 +264,18 @@ def test_mail_mass_mode_compose_with_mso(self):
 @tagged("mail_composer", "-at_install", "post_install")
 class TestMailComposerUI(MailCommon, HttpCase):
 
+    @classmethod
+    def setUpClass(cls):
+        super(TestMailComposerUI, cls).setUpClass()
+        cls.env['mail.alias.domain'].create({'name': 'example.com'})
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'test customer',
+            'email': 'dummy@example.com'
+        })
+        cls.quotation = cls.env['sale.order'].create({
+            'partner_id': cls.partner.id,
+        })
+
```

Comment:
`mail` installed doesn't necessarily mean `sale` is installed. The other-way yes. Therefore this should be moved to `sale` module.